pkgs/development/libraries/libgcrypt/1.5.nix

```diff
@@ -10,6 +10,14 @@ stdenv.mkDerivation rec {
     sha256 = "0ydy7bgra5jbq9mxl5x031nif3m6y3balc6ndw2ngj11wnsjc61h";
   };
 
+  patches = stdenv.lib.optionals stdenv.isDarwin [
```

Comment:
Not everybody agrees on this, but I personally prefer to avoid conditional patches if they are generic. Making it much more noticeable for everybody if a patch does not apply anymore.